docs/examples.rst

```diff
@@ -293,3 +293,45 @@ prettyprints the single dimension dict:
             "section_pk": 707,
         },
     }
+
+How to Data Upload
+^^^^^^^^^^^^^^^^^^^
+
+Currently the API supports the ability to search for data groups one owns, check the processing status, and upload valid DataFrames. One must still upload data through the UI - specifying the schema. The API is meant to programmatically refresh the data.
+
+..  code::
+
+    for data_group in dm.get_data_groups():
+        print(data_group)
+
+Alternatively, one can fetch a data group by its ID and view it's status:
+
+..  code::
+
+    dg = dm.get_data_group_by_id(1012)
+    dg.get_current_status()
+
+To view the columns of the data group, and hence the schema, to verify the type of data we wish to reupload:
```

Comment:
maybe rephrase?
To view the columns/schema of the data group in order to verify the type of data we wish to re-upload:

Reply:
thx